wordpress-mobile/WordPress-iOS
# Adds logic to determine when to show the post signup interstitial

Fixes #12877 and #12916

To test:
1. Launch the app
2. If you're logged in, log out
3. Login to an account that has no sites
3.a Sign up for a new account
4. You will be shown the epilogue views
5. Tap next
6. You should be displayed the interstitial
7. Relaunch the app, you should not be displayed the interstitial

PR submission checklist:
